### PR TITLE
Restore app dir and move session debugger to /debugger

### PR DESCRIPTION
**Pull request by alebedev**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3893?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #3879** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3879?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3893** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3893?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3893?utm_source=stack-comment).

File: apps/store/src/app/debugger/actions.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use server'
```
Comment (gustaveen) [inline apps/store/src/app/debugger/actions.ts:1]: I like that we're explicit here, but just wondering, it would be server rendered bu default right?

**Review by gustaveen — APPROVED** [review]
👍

File: apps/store/src/app/debugger/page.tsx
Comment (guilhermespopolin) [inline apps/store/src/app/debugger/page.tsx:1]: Should we migrate the styles to vanilla-extract here and transform this into a RSC?

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/app/debugger/actions.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/app/debugger/actions.ts:1]: This is yet another type of module - server action https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations

You're right, for RSC we don't need add `'use server'`

File: apps/store/src/app/debugger/page.tsx
Comment (alebedev) [reply apps/store/src/app/debugger/page.tsx:1]: Yes, in the follow-up steps

File: apps/store/src/app/debugger/actions.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/app/debugger/actions.ts:1]: Gotcha! 🙌

**Pull request merged**